CHANGELOG.md

```diff
@@ -1,208 +1,284 @@
-### 4.2.0 - 2022-01-21
+# 5.0.0 (2022-04-29)
```

Comment:
@benschwarz I added the changes for `5.0.0` here, you might have to update the date if that doesn't end up being the release date.

i also formatted the entire file so it's consistent and categorises changes for easier reading. you might want to have a scan if everything is appropriately categorised. 😅

Reply:
Tomorrow is the day!! 🕐